test/results/PlatinumSample_NA12878.NGS_DNA_Test_v1.coveragePerBase.selection_True.txt

```diff
@@ -1,100 +1,100 @@
-10	2	197559	AC079779.7	33.0	CDS	1
-11	2	197560	AC079779.7	33.0	CDS	1
-12	2	197561	AC079779.7	33.0	CDS	1
-13	2	197562	AC079779.7	33.0	CDS	1
-14	2	197563	AC079779.7	33.0	CDS	1
-15	2	197564	AC079779.7	36.0	CDS	1
-16	2	197565	AC079779.7	36.0	CDS	1
-17	2	197566	AC079779.7	36.0	CDS	1
-18	2	197567	AC079779.7	36.0	CDS	1
-19	2	197568	AC079779.7	31.0	CDS	1
-20	2	197569	AC079779.7	31.0	CDS	1
-21	2	197570	AC079779.7	31.0	CDS	1
-22	2	197571	AC079779.7	31.0	CDS	1
-23	2	197572	AC079779.7	31.0	CDS	1
-24	2	197573	AC079779.7	31.0	CDS	1
-25	2	197574	AC079779.7	31.0	CDS	1
-26	2	197575	AC079779.7	31.0	CDS	1
-27	2	197576	AC079779.7	31.0	CDS	1
-28	2	197577	AC079779.7	31.0	CDS	1
-29	2	197578	AC079779.7	31.0	CDS	1
-30	2	197579	AC079779.7	31.0	CDS	1
-31	2	197580	AC079779.7	31.0	CDS	1
-32	2	197581	AC079779.7	31.0	CDS	1
-33	2	197582	AC079779.7	31.0	CDS	1
-34	2	197583	AC079779.7	31.0	CDS	1
-35	2	197584	AC079779.7	28.0	CDS	1
-36	2	197585	AC079779.7	28.0	CDS	1
-37	2	197586	AC079779.7	28.0	CDS	1
-38	2	197587	AC079779.7	28.0	CDS	1
-39	2	197588	AC079779.7	28.0	CDS	1
-40	2	197589	AC079779.7	28.0	CDS	1
-41	2	197590	AC079779.7	28.0	CDS	1
-42	2	197591	AC079779.7	28.0	CDS	1
-43	2	197592	AC079779.7	28.0	CDS	1
-44	2	197593	AC079779.7	27.0	CDS	1
-45	2	197594	AC079779.7	26.0	CDS	1
-46	2	197595	AC079779.7	26.0	CDS	1
-47	2	197596	AC079779.7	26.0	CDS	1
-48	2	197597	AC079779.7	25.0	CDS	1
-49	2	197598	AC079779.7	25.0	CDS	1
-52329	Y	2803782	ZFY	0.0	CDS	1
-52330	Y	2803783	ZFY	0.0	CDS	1
-52331	Y	2803784	ZFY	0.0	CDS	1
-52332	Y	2803785	ZFY	0.0	CDS	1
-52333	Y	2803786	ZFY	0.0	CDS	1
-52334	Y	2803787	ZFY	0.0	CDS	1
-52335	Y	2803788	ZFY	0.0	CDS	1
-52336	Y	2803789	ZFY	0.0	CDS	1
-52337	Y	2803790	ZFY	0.0	CDS	1
-52338	Y	2803791	ZFY	0.0	CDS	1
-52339	Y	2803792	ZFY	0.0	CDS	1
-52340	Y	2803793	ZFY	0.0	CDS	1
-52341	Y	2803794	ZFY	0.0	CDS	1
-52342	Y	2803795	ZFY	0.0	CDS	1
-52343	Y	2803796	ZFY	0.0	CDS	1
-52344	Y	2803797	ZFY	0.0	CDS	1
-52345	Y	2803798	ZFY	0.0	CDS	1
-52346	Y	2803799	ZFY	0.0	CDS	1
-52347	Y	2803800	ZFY	0.0	CDS	1
-52348	Y	2803801	ZFY	0.0	CDS	1
-52349	Y	2803802	ZFY	0.0	CDS	1
-52350	Y	2803803	ZFY	0.0	CDS	1
-52351	Y	2803804	ZFY	0.0	CDS	1
-52352	Y	2803805	ZFY	0.0	CDS	1
-52353	Y	2803806	ZFY	0.0	CDS	1
-52354	Y	2803807	ZFY	0.0	CDS	1
-52355	Y	2803808	ZFY	0.0	CDS	1
-52356	Y	2803809	ZFY	0.0	CDS	1
-52357	Y	2803810	ZFY	0.0	CDS	1
-52358	Y	2803811	ZFY	0.0	CDS	1
-52359	Y	2803812	ZFY	0.0	CDS	1
-52360	Y	2803813	ZFY	0.0	CDS	1
-52361	Y	2803814	ZFY	0.0	CDS	1
-52362	Y	2803815	ZFY	0.0	CDS	1
-52363	Y	2803816	ZFY	0.0	CDS	1
-52364	Y	2803817	ZFY	0.0	CDS	1
-52365	Y	2803818	ZFY	0.0	CDS	1
-52366	Y	2803819	ZFY	0.0	CDS	1
-52367	Y	2803820	ZFY	0.0	CDS	1
-52368	Y	2803821	ZFY	0.0	CDS	1
-52369	Y	2803822	ZFY	0.0	CDS	1
-52370	Y	2803823	ZFY	0.0	CDS	1
-52371	Y	2803824	ZFY	0.0	CDS	1
-52372	Y	2803825	ZFY	0.0	CDS	1
-52373	Y	2803826	ZFY	0.0	CDS	1
-52374	Y	2803827	ZFY	0.0	CDS	1
-52375	Y	2803828	ZFY	0.0	CDS	1
-52376	Y	2803829	ZFY	0.0	CDS	1
-52377	Y	2803830	ZFY	0.0	CDS	1
-52378	Y	2803831	ZFY	0.0	CDS	1
+1	2	197550	AC079779.7	33	CDS	1		
```

Comment:
Klopt het dat er 2 extra tabs toegevoegd moeten worden aan elke regel? Ik zie geen extra kolommen in de header...

Reply:
lijkt me niet nee :/